baybe/targets/numerical.py

```diff
@@ -353,26 +354,32 @@ def normalize_sigmoid(
 
         Args:
             name: The name of the target.
-            anchors: See :class:`baybe.transformations.core.SigmoidTransformation`.
+            anchors: See :class:`baybe.transformations.basic.SigmoidTransformation`.
 
         Returns:
             The target with applied sigmoid transformation.
         """
         return NumericalTarget(name, SigmoidTransformation.from_anchors(anchors))
 
     @property
-    def is_normalized(self) -> bool:
+    def is_normalized(self) -> UncertainBool:
```

Comment:
`normalized` now has the option to specify `codomain: bool` but the property here is hard-coded for the image. What do we do?
* Keep it and ignore codomain
* add second property
* turn into method, offering the same option

Reply:
done